test/scripts/e2e_subs/v26/teal-v3-only.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+date '+teal-v3-only start %Y%m%d_%H%M%S'
+
+set -e
+set -x
+set -o pipefail
+export SHELLOPTS
+
+WALLET=$1
+
+gcmd="goal -w ${WALLET}"
+
+ACCOUNT=$(${gcmd} account list|awk '{ print $3 }')
+
+# prints:
+# Created new account with address UCTHHNBEAUWHDQWQI5DGQCTB7AR4CSVNU5YNPROAYQIT3Y3LKVDFAA5M6Q
+ACCOUNTB=$(${gcmd} account new|awk '{ print $6 }')
+
+cat >${TEMPDIR}/true.teal<<EOF
+#pragma version 3
+int 1
+return
```

Comment:
don't need `return`

Reply:
Right this is a remnant from the teal-v2 test, where I wanted to be clear that I had v3 code (where return was introduced). I think it still makes *some* sense here because of that, but the one below should be changed to some v4 opcode.

Reply:
Nope, I'm wrong.  return was in v2.  So it's entirely extraneous. I guess I got in the habit of doing that in tests when I wasn't sure of the stack.